File: Android/Martinshare/app/src/main/java/mobile/martinshare/com/martinshare/API/protocols/EintragenProtocol.java
package mobile.martinshare.com.martinshare.API.protocols;

/**
 * Created by Modestas Valauskas on 06.06.2015.
 */
public interface EintragenProtocol {
    public void startedEintragen();
    public void eingetragen();
    public void isNotLoggedIn();
    public void noInternet();
    public void unknownError(final String error) ;
}

File: iOS/Martinshare iOS/Martinshare/EintragenProtocol.swift

//
//  EintragenProtocol.swift
//  Martinshare
//
//  Created by Modestas Valauskas on 30.05.15.
//  Copyright (c) 2015 ModestasV Studios. All rights reserved.
//

import Foundation

protocol EintragenProtocol {
    
    func startedConnection()
    func notLoggedIn()
    //Ich werde die Eintraege aktualisieren und dem nutzer anzeigen "BITTE VERSUCHE ES SPÄTER NOCHMAL"
    func unknownError()
    func noInternet()
    func aktualisiert()
}